### DISCUSSION TITLE
Import Error | Obfuscate problem case

### DISCUSSION THREAD

**Discussion by dmkn96** [dashingsoft/pyarmor#1759] [Q&A] [answered]
Hello! We have encountered a problem with obfuscating our code on `Python version 3.7`, 
and subsequently faced an issue where this code needs to be read on devices with Python 
versions higher than `3.7`. In such cases, we get import errors. 

I understand that after obfuscation, `the code can only be interpreted or read by the same version of Python.` 
Are there any ways to circumvent this issue without `creating separate obfuscated files for each Python version?`
Is it possible to encrypt the code once and then read it using different .pyc 
files depending on the Python version the user has? 

Please advise on what would be the most correct and convenient approach?
Thank you~~

**Comment by jondy** [comment, marked as answer]
Here it's the solution for this issue now
https://pyarmor.readthedocs.io/en/latest/tutorial/advanced.html#obfuscating-scripts-for-multiple-python-versions

**Reply by dmkn96** [reply]
Thank you~
